e2e-testing/docker-compose-e2e-test.yml

```diff
@@ -194,6 +194,7 @@ services:
     environment:
       - WS_PORT=3003
       - COMLINK_URL=host.docker.internal:3002
+      - INDEXER_LEVEL_GEOBLOCKING_ENABLED=false
```

Comment:
instead of setting this to false everywhere, can we set it to false in `.env` and set it to true in `.env.production`, so it's always true in production but off in dev and tests? We can do this for socks and comlink

Reply:
done